oml/lightning/callbacks/metric.py

```diff
@@ -72,6 +89,30 @@ def on_validation_batch_end(
             if is_last_expected_batch:
                 self.calc_and_log_metrics(pl_module)
 
+    def _log_images(self, pl_module: pl.LightningDataModule) -> None:
+        if not isinstance(self.metric, EmbeddingMetrics):
+            return
+
+        for metric_name in flatten_dict(self.metric.metrics):  # type: ignore
+            if any([ignore in metric_name for ignore in self.image_metrics_to_ignore]):
+                continue
+            if self.log_only_main_category and not metric_name.startswith(OVERALL_CATEGORIES_KEY):
```

Comment:
is it possible?

Reply:
We actually have no guaranties for this to be always impossible. This check is pretty fast anyway, so why not to do it.